File: samples/client/petstore/java/src/main/java/org/openapijsonschematools/client/paths/FakemultipleRequestBodyContentTypes.java
package org.openapijsonschematools.client.paths;

import org.openapijsonschematools.client.apiclient.ApiClient;
import org.openapijsonschematools.client.configurations.ApiConfiguration;
import org.openapijsonschematools.client.configurations.SchemaConfiguration;

public class FakemultipleRequestBodyContentTypes extends ApiClient {
    protected static final String path = "/fake/multipleRequestBodyContentTypes/";

    public FakemultipleRequestBodyContentTypes(ApiConfiguration apiConfiguration, SchemaConfiguration schemaConfiguration) {
        super(apiConfiguration, schemaConfiguration);
    }
}

File: samples/client/petstore/kotlin/src/main/kotlin/org/openapijsonschematools/client/apis/paths/Fakemultiplerequestbodycontenttypes.kt
package org.openapijsonschematools.client.apis.paths;

import org.openapijsonschematools.client.apiclient.ApiClient;
import org.openapijsonschematools.client.configurations.ApiConfiguration;
import org.openapijsonschematools.client.configurations.SchemaConfiguration;
import org.openapijsonschematools.client.paths.fakemultiplerequestbodycontenttypes.FakemultiplerequestbodycontenttypesPost;

public class Fakemultiplerequestbodycontenttypes extends ApiClient implements
    FakemultiplerequestbodycontenttypesPost.PostOperation
{
    public Fakemultiplerequestbodycontenttypes(ApiConfiguration apiConfiguration, SchemaConfiguration schemaConfiguration) {
        super(apiConfiguration, schemaConfiguration);
    }
}